### PR TITLE
Deactivate transfer card

### PR DESCRIPTION
**Pull request by enjeyw**
**Describe the Pull Request**

App side capacity to deactivate a transfer card. 

**Todo**

- [ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [ ] Request review from relevant @person or team
- [ ] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [ ] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Ability to deactivate cards](https://trello.com/c/AEsSTOK1/143-ability-to-deactivate-cards)

File: app/migrations/versions/afdbf2c8d633_.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"""empty message
```
Comment (michielderoos) [inline app/migrations/versions/afdbf2c8d633_.py:1]: Nit: Migration message should be populated

File: app/server/api/transfer_card_api.py [hunk 2]
```diff
@@ -135,13 +139,17 @@ def put(self, public_serial_number, nfc_serial_number):
             }
             return make_response(jsonify(response_object)), 404
 
-        if not transfer_card.transfer_account:
-            response_object = {
-                'message': f'Card {transfer_card.id} not bound to transfer account',
-            }
-            return make_response(jsonify(response_object)), 400
+        if disable:
```
Comment (michielderoos) [inline app/server/api/transfer_card_api.py:142]: Should we `400` if we try to disable the same card twice? Right now it's not allowed on the FE, but you can hit this API over and over

File: app/client/sagas/transferCardSagas.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    EditTransferCardActionTypes.EDIT_TRANSFER_CARD_REQUEST,
+    EditTransferCardPayload
+  >
+) {
+  try {
+    const load_result = yield call(editTransferCardAPI, action.payload);
+
+    yield put(EditTransferCardAction.editTransferCardSuccess());
+
+    yield put(LoadUserAction.loadUserRequest({ path: action.payload.userId }));
```
Comment (michielderoos) [inline app/client/sagas/transferCardSagas.ts:29]: `action.payload.userId` is undefined, so this makes the FE call `/api/v1/user/?org=1` (with no user ID). This makes it load near-indefinitely on deployments with a lot of users (which is an out-of-scope problem)

File: app/client/components/user/EditUserForm.tsx [hunk 4]
```diff
@@ -264,13 +267,34 @@ class EditUserForm extends React.Component<
...
-                    />
+                    {/* // @ts-ignore */}
+
+                    <div style={{ display: "flex", alignItems: "flex-end" }}>
+                      <QrReadingModal
+                        updateData={(data: string) =>
+                          this.setSerialNumber(data)
+                        }
+                      />
+                      {selectedUser.public_serial_number ? (
```
Comment (michielderoos) [inline app/client/components/user/EditUserForm.tsx:288]: Should there be some visual cue that the card is already disabled?

**Review by michielderoos — CHANGES_REQUESTED** [review]
Looks good, just a few little changes!

File: app/migrations/versions/afdbf2c8d633_.py (hunk 1 above)
Comment (enjeyw) [reply app/migrations/versions/afdbf2c8d633_.py:1]: Hahaha whoops it's adctually on the line below lol!

File: app/client/components/user/EditUserForm.tsx (hunk 4 above)
Comment (enjeyw) [reply app/client/components/user/EditUserForm.tsx:288]: There is already - turns red (see line 275 above)

File: app/server/api/transfer_card_api.py (hunk 2 above)
Comment (enjeyw) [reply app/server/api/transfer_card_api.py:142]: done :)

File: app/client/sagas/transferCardSagas.ts (hunk 3 above)
Comment (michielderoos) [reply app/client/sagas/transferCardSagas.ts:29]: This is still an issue

File: app/client/components/user/EditUserForm.tsx (hunk 4 above)
Comment (michielderoos) [reply app/client/components/user/EditUserForm.tsx:290]: Darn this colourblindness! 👀

**Review by michielderoos — CHANGES_REQUESTED** [review]
Tests were broken (I think I fixed them though). Still have one [little issue](https://github.com/teamsempo/SempoBlockchain/pull/602#discussion_r587725643), but looks good after that

File: app/client/sagas/transferCardSagas.ts (hunk 3 above)
Comment (enjeyw) [reply app/client/sagas/transferCardSagas.ts:29]: oh sorry i missinterpreted as leaving this for different pr!

Comment (enjeyw) [reply app/client/sagas/transferCardSagas.ts:29]: fixed now (in the edit component itself)

**Review by michielderoos — APPROVED** [review]
🚀

File: app/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "SempoBlockchain",
-  "version": "1.3.14",
```
Comment (michielderoos) [inline app/package.json:3]: Pi!

**Pull request merged**